specs/engage-digital_openapi3.yaml

```diff
@@ -2189,6 +2198,15 @@ paths:
           schema:
             format: int32
             type: integer
+        - description: >-
+            true or false, that indicates if the custom field is in read-only
+            (default: true).

```

Comment:
default false no ?

Reply:
Indeed, I have fixed this and regenerated the json files